libs/ledger-live-common/src/families/ethereum/modules/erc1155.ts

```diff
@@ -146,7 +146,8 @@ function serializeTransactionData(
   const from = eip55.encode(account.freshAddress);
   const to = eip55.encode(transaction.recipient);
   const tokenIds = transaction.tokenIds || [];
-  const quantities = transaction.quantities?.map((q) => q.toFixed()) || [];
+  const quantities =
+    transaction.quantities?.map((q) => q?.toFixed() || "0") || [];
```

Comment:
Possible alternative in case we prefer filtering empty values instead of adding a "0" default value.
I have no idea so I let you decide.
```suggestion
const quantities = transaction.quantities?.filter(Boolean).map((q) => q.toFixed()) || [];
```

Reply:
I think in this specific case the `"0"` fallback actually makes sense cause we're about to encode the transfer function with the correct ABI, which means we are falling back to:

`safeTransferFrom(0x123, 0x456, 1, 0, null)` where the 4th parameter is the quantity, which means we are still encoding a correct contract interaction, just sending 0 NFT.
I'm not sure how it would behave with a null/undefined instead 🤔